hadoop-ozone/dist/pom.xml

```diff
@@ -29,7 +29,7 @@
     <file.encoding>UTF-8</file.encoding>
     <downloadSources>true</downloadSources>
     <docker.ozone-runner.version>20210329-1</docker.ozone-runner.version>
-    <docker.ozone-testkr5b.image>apache/ozone-testkrb5:20210419-1</docker.ozone-testkr5b.image>
+    <docker.ozone-testkr5b.image>apache/ozone-testkrb5:latest</docker.ozone-testkr5b.image>
```

Comment:
It seems that we missed creating a tag for the changes. Based on the commit date in ozone-docker-testkrb5's master branch, I have tagged that version as 20211102-1.
It should be available at some point in time later on, and I think we should switch to that tag instead of latest, as latest might mess up with principals and keytabs if there is an untagged change on the master in the testkrb branch.

Let's switch to the tag, and wait for that to be ready for use, before merging this change in.

Reply:
Thanks for tagging the new version, it is much better this way. It is available by now, so I pushed the changes.